meilisearch_python_async/models/settings.py

```diff
@@ -15,6 +15,10 @@ class TypoTolerance(CamelBase):
     min_word_size_for_typos: Optional[MinWordSizeForTypos] = None
 
 
+class Faceting(CamelBase):
+    max_values_per_facet: Optional[int] = None
```

Comment:
This one can be required because if `Faceting` is used by itself it doesn't make since for `max_values_per_facet` to be `None`, and the `None` case will be handled in `MeiliSearchSettings`

```suggestion
    max_values_per_facet: int
```